translations/base-fi.yaml

```diff
@@ -934,65 +904,54 @@ keybindings:
         menuOpenShop: Päivitykset
         menuOpenStats: Tilastot
         toggleHud: Vaihda valikon näkyvyys
-        toggleFPSInfo: Vaihda FPS and Virheenkorjaus tiedot
+        toggleFPSInfo: Vaihda FPS- and virheenkorjaustiedot
         switchLayers: Vaihda tasoa
         exportScreenshot: Vie koko tukikohta kuvana
-        belt: Liukuhihna
+        belt: Kuljetin
         underground_belt: Tunneli
-        miner: Kaivaja
+        miner: Poimija
         cutter: Leikkuri
         rotater: Kääntäjä
         stacker: Pinoaja
         mixer: Värinsekoittaja
-        painter: Värjääjä
+        painter: Maalain
         trash: Roskakori
         wire: Johto
         pipette: Pipetti
         rotateWhilePlacing: Käännä
-        rotateInverseModifier: "Modifier: Käännä sen sijaan vastapäivään"
+        rotateInverseModifier: "Muutos: Käännä vastapäivään"
         cycleBuildingVariants: Valitse muotoja
         confirmMassDelete: Poista alue
         pasteLastBlueprint: Liitä viimeisin piirustus
-        cycleBuildings: Valitse Rakennuksia
-        lockBeltDirection: Ota liukuhihnasuunnittelija käyttöön
-        switchDirectionLockSide: "Planner: Muuta sivua"
+        cycleBuildings: Valitse rakennuksia
+        lockBeltDirection: Ota kuljetinsuunnittelija käyttöön
+        switchDirectionLockSide: "Suunnittelija: Muuta sivua"
         massSelectStart: Pidä pohjassa ja raahaa aloittaaksesi
         massSelectSelectMultiple: Valitse useita alueita
         massSelectCopy: Kopioi alue
         massSelectCut: Leikkaa alue
         placementDisableAutoOrientation: Poista automaattinen suunta käytöstä
         placeMultiple: Pysy sijoittamistilassa
-        placeInverse: Käännä automaattinen hihnan suunta päinvastoin
+        placeInverse: Käännä automaattinen kuljettimen suunta päinvastoin
         menuClose: Sulje valikko
-        balancer: Balancer
-        storage: Storage
-        constant_signal: Constant Signal
-        logic_gate: Logic Gate
-        lever: Switch (regular)
-        filter: Filter
-        wire_tunnel: Wire Crossing
-        display: Display
-        reader: Belt Reader
-        virtual_processor: Virtual Cutter
-        transistor: Transistor
-        analyzer: Shape Analyzer
-        comparator: Compare
-        item_producer: Item Producer (Sandbox)
```

Comment:
Why is this whole part missing???

Reply:
Below it

Reply:
I have no idea. Maybe I messed it up originally. It's now back there. Hope it helps with the errors too

Reply:
YAY! It's error free if I read the log correctly. Finally I can go to sleep